.napari-hub/DESCRIPTION.md

```diff
@@ -31,3 +31,18 @@ We developed `btrack` for cell tracking in time-lapse microscopy data.
 ## associated plugins
 
 * [napari-arboretum](https://www.napari-hub.org/plugins/napari-arboretum) - Napari plugin to enable track graph and lineage tree visualization.
+
+## Installation
+
+To install the `napari` plugin associated with `btrack` run the command.
+
+```sh
+pip install btrack[napari]
+```
+
+If working on M1 Mac/Apple Silicon/osx-arm64 make sure to install the following
+packages from `conda-forge`.
+
+```sh
+conda install -c conda-forge cvxopt pyqt
```

Comment:
Should this happen before the pip install?

Reply:
I've just tried and it makes no difference. I feel like afterwards makes more sense from the point of view of "in addition, you will need to..."

Reply:
Cool - OK. Wasn't totally sure how conda and pip play together in such situations.

Reply:
It all comes down to where they are on your `PATH`.